frontend/packages/console-shared/src/selectors/machine.ts

```diff
@@ -0,0 +1,11 @@
+import * as _ from 'lodash-es';
+
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { getName } from './common';
+
+export const getMachineRole = (machine) =>
+  _.get(machine, ['metadata', 'labels', 'machine.openshift.io/cluster-api-machine-role']);
```

Comment:
We already have a helper for this here:

https://github.com/openshift/console/blob/master/frontend/public/components/machine.tsx#L26

It looks like the label is different though. We should check what's correct and only have this helper in once place.

Reply:
Looking here https://github.com/openshift/machine-api-operator it seems that the label has been changed to `machine.openshift.io`

Reply:
> Looking here https://github.com/openshift/machine-api-operator it seems that the label has been changed to `machine.openshift.io`

@jtomasek We need to evaluate whether this was shipped as a bug in 4.1. If it was, we should open a Bugzilla to fix it in 4.1.z.